test/runtest.py

```diff
@@ -378,6 +378,28 @@ def f():
     assert ray.get(f.remote()) == ((3, "string1", Bar.__name__), "string2")
 
 
+def test_serialization_final_fallback(ray_start):
+    pytest.importorskip('catboost')
```

Comment:
It'd be ideal to run this test all of the time and not skip it. Does this run in Travis?